inc/class-wpseo-admin-bar-menu.php

```diff
@@ -214,6 +216,50 @@ protected function add_root_menu( WP_Admin_Bar $wp_admin_bar ) {
 		}
 	}
 
+	/**
+	 * Adds the focus keyword item.
+	 *
+	 * @param WP_Admin_Bar $wp_admin_bar Admin bar instance to add the menu to.
+	 *
+	 * @return void
+	 */
+	protected function add_focus_keyword_submenu( WP_Admin_Bar $wp_admin_bar ) {
+		$post_id = $this->get_singular_post_id();
+		if ( ! $post_id ) {
+			return;
+		}
+
+		$focus_keyword = $this->get_post_focus_keyword( $post_id );
+		if ( empty( $focus_keyword ) ) {
+			$focus_keyword = '<em>not set</em>';
```

Comment:
This is not translatable.

Also wondering if when no keyphrase the whole menu items should just say "No focus keyphrase set" instead of the current two lines text:

Focus keyphrase for this post:
not set

<img width="281" alt="Screenshot 2019-10-09 at 09 51 39" src="https://user-images.githubusercontent.com/1682452/66462213-eb885180-ea7a-11e9-90e5-ab3282aadd51.png">

Reply:
Jup, good idea, implementing.